core/App/components/inputs/PinInput.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { Keyboard, StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { CodeField, useClearByFocusCell } from 'react-native-confirmation-code-field'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+import { useTheme } from '../../contexts/theme'
+
+const CELL_COUNT = 6
```

Comment:
I think we're using camel case for consts except for keychain/async storage keys.

Reply:
Moved to constants file